CommonAssemblyInfo.cs

```diff
@@ -1,4 +1,4 @@
 using System.Reflection;
 
-[assembly: AssemblyVersion("6.5.0")]
-[assembly: AssemblyFileVersion("6.5.0")]
+[assembly: AssemblyVersion("6.90.0")]
```

Comment:
Ah, I misunderstood what Paul was asking. OK, should be good now but I still see lots of commits that aren't mine.